src/commands/runDraftTool/runDraftWorkflow.ts

```diff
@@ -0,0 +1,471 @@
+         (rg) => rgToItem(rg).label === rgPick.label
+      );
+   }
+
+   public shouldPrompt(wizardContext: WizardContext): boolean {
+      return true;
+   }
+}
+
+class PromptACRSelection extends AzureWizardPromptStep<WizardContext> {
+   constructor(private az: AzApi) {
+      super();
+   }
+
+   public async prompt(wizardContext: WizardContext): Promise<void> {
+      if (wizardContext.clusterSubscription === undefined) {
+         throw Error('ACR Subscription is undefined');
+      }
+      if (wizardContext.acrResourceGroup === undefined) {
+         throw Error('ACR Resource Group is undefined');
+      }
+
+      const registries = getAysncResult(
+         this.az.listContainerRegistries(
+            wizardContext.clusterSubscription,
+            wizardContext.acrResourceGroup
+         )
+      );
+
+      const registryToItem = (r: RegistryItem) => ({
+         label: r.registry.name || ''
+      });
+
+      const registryPick = await wizardContext.ui.showQuickPick(
+         sort(getAsyncOptions(registries, registryToItem)),
+         {
+            ignoreFocusOut,
+            stepName: 'Registry',
+            placeHolder: 'Registry',
+            noPicksMessage: 'No Registries found'
+         }
+      );
+
+      wizardContext.registry = (await registries).find(
+         (r) => registryToItem(r).label === registryPick.label
+      );
+   }
+
+   public shouldPrompt(wizardContext: WizardContext): boolean {
+      return true;
+   }
+}
+
+class PromptACRRegistrySelection extends AzureWizardPromptStep<WizardContext> {
+   constructor(private az: AzApi) {
+      super();
+   }
+
+   public async prompt(wizardContext: WizardContext): Promise<void> {
+      if (wizardContext.clusterSubscription === undefined) {
+         throw Error('ACR Subscription is undefined');
+      }
+      if (wizardContext.registry === undefined) {
+         throw Error('ACR Registry is undefined');
+      }
+
+      const repositories = getAysncResult(
+         this.az.listRegistryRepositories(
+            wizardContext.clusterSubscription,
+            wizardContext.registry
+         )
+      );
+      const repositoryToItem = (r: RepositoryItem) => ({
+         label: r.repositoryName
+      });
+
+      const repositoryPick = await wizardContext.ui.showQuickPick(
+         sort(getAsyncOptions(repositories, repositoryToItem)),
+         {
+            ignoreFocusOut,
+            stepName: 'Repository',
+            placeHolder: 'Repository',
+            noPicksMessage: 'No Repositories found'
+         }
+      );
+
+      wizardContext.acrRepository = (await repositories).find(
+         (r) => repositoryToItem(r).label === repositoryPick.label
+      );
+   }
+
+   public shouldPrompt(wizardContext: WizardContext): boolean {
+      return true;
+   }
+}
+
+class PromptGitHubBranchSelection extends AzureWizardPromptStep<WizardContext> {
+   public async prompt(wizardContext: WizardContext): Promise<void> {
```

Comment:
more of a question than feedback but where does wizardContext.destination get set? Is this something users are expected to configure before using the extension?